apps/etterlatte-saksbehandling-ui/client/src/components/nyoppgavebenk/Oppgavelistafiltre.tsx

```diff
@@ -82,13 +82,23 @@ function filtrerOppgaveStatus(
 }
 
 export type OppgavetypeFilterKeys = Oppgavetype | visAlle
-export const OPPGAVETYPEFILTER: Record<OppgavetypeFilterKeys, string> = {
-  visAlle: 'Vis alle',
-  FOERSTEGANGSBEHANDLING: 'Førstegangsbehandling',
-  REVURDERING: 'Revurdering',
-  HENDELSE: 'Hendelse',
-  MANUELT_OPPHOER: 'Manuelt opphør',
-  EKSTERN: 'Ekstern',
+export const OPPGAVETYPEFILTER: {
```

Comment:
Hvorfor er denne endret, den skal være en record av typen `OppgavetypeFilterKeys` for å sikre at typene fra `Oppgavetype` som brukes i oppgaven er konsistent.

Denne skulle vært slik:
```suggestion
export const OPPGAVETYPEFILTER: Record<OppgavetypeFilterKeys, string> = {
  visAlle: 'Vis alle',
  FOERSTEGANGSBEHANDLING: 'Førstegangsbehandling',
  REVURDERING: 'Revurdering',
  HENDELSE: 'Hendelse',
  MANUELT_OPPHOER: 'Manuelt opphør',
  EKSTERN: 'Ekstern',
  ATTESTERING: 'Attestering'
}
```

Reply:
også må `Oppgavetype` i oppgaverny.ts utvides med attestering.